app/components/formatted_text.js

```diff
@@ -18,14 +17,18 @@ class FormattedText extends React.PureComponent {
         defaultMessage: '',
     };
 
+    static contextTypes = {
+        intl: intlShape.isRequired,
```

Comment:
I want to start to get rid of the context per https://reactjs.org/docs/context.html

Reply:
I am thinking about that as well since context is global and yet we're doing it on a per component level.  Maybe let's experiment into it one time.